File: backend/src/main/java/com/prefin/domain/entertainment/Quiz.java
package com.prefin.domain.entertainment;

import lombok.*;

import javax.persistence.*;

@Entity
@NoArgsConstructor
@AllArgsConstructor
@Getter
@Builder
public class Quiz {
    @Id @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(columnDefinition = "VARCHAR(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci")
    private String question;

    private int answer;

    @Column(columnDefinition = "VARCHAR(255) CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci")
    private String description;
}

File: mobile/Prefin/app/src/main/java/com/prefin/model/dto/Quiz.kt
package com.prefin.model.dto

data class Quiz(
    var answer: Boolean,
    val description: String = "퀴즈가 없어요",
    val id: Int,
    val question: String = "더 이상 퀴즈가 없습니다."
){
    constructor() : this(false, "퀴즈가 없어요", 0, "퀴즈가 없어요")
}